src/mysql/populate.sql

```diff
@@ -13,15 +13,15 @@ VALUES
   ('ITI'),
   ('IMT');
 
-INSERT INTO User (name, email, major_id, is_mentor) 
+INSERT INTO User (fname, lname, username, email, major_id, is_mentor) 
```

Comment:
if you make the change from create.sql be sure to update this too

Reply:
Yup!